src/mmw/js/src/modeling/models.js

```diff
@@ -45,6 +50,11 @@ var ProjectModel = Backbone.Model.extend({
         // hard-coded here.
         this.set('model_package', 'tr-55');
         this.set('taskModel', new Tr55TaskModel());
+        this.set('uid', App.user.get('uid'));
+
+        this.get('scenarios').on('add', this.addIdsToScenarios, this);
```

Comment:
Consider using `listenTo` instead of `on` so undelegating the events is handled automatically.